src/info_html.R

```diff
@@ -54,13 +54,39 @@ renderMethodsHtml <- function() {
         tags$a("Verity et al (2020).", href = urls$verity_etal_2020),
         "We chose this study over US CDC reports because this study is larger and more thorough. We do not account for differences between Chinese population and US population."
       ),
+      tags$li("Estimations for risk adjustment based on gender are from this preprint by", 
+              tags$a("Caramelo et al (2020).", href = urls$caramelo_etal_2020)
+      ),
       tags$li("Estimations of risk factors associated with underlying medical conditions were obtained from",
               tags$a("China CDC weekly, 2020 Vol No.2", href = urls$ccdc_vol2_2020), ", ",
+              tags$a("Simonnet et al (2020)", href = urls$simonnet_ettal), ", ",
               tags$a("Killerby et al (2020)", href = urls$cdc_hosp_June2020), ", and ",
-              tags$a("OpenSafely", href = urls$open_safely),
-              "and gender from this preprint by", 
-              tags$a("Caramelo et al (2020).", href = urls$caramelo_etal_2020)
-      )
+              tags$a("OpenSafely. ", href = urls$open_safely), "Mutually adjusted odds ratios ", 
+              "(adjusted for other underlying conditions as well as age and gender) are used when available. ",
+              "Otherwise, unadjusted odds ratios are used but the user's risk is adjusted only for the two highest risk conditions ",
+              "inputted by the user. ", "These adjustment odds ratios will be updated as more research becomes available.",
+              "The following describes the current risk adjustment procedure:",
+              
+              tags$ul(tags$li("To adjust hospitalization risk, we use mutually adjusted odds ratios from Killerby et al. ",
+                              "This paper does not report odds ratios for 'immunocompromised condition' or ", 
+                              "'other chronic disease', so for these two categories we use unadjusted odds ratios from CDCC ",
+                              "and only adjust for these two categories if no other conditions are inputted by the user. ",
+                              "When the reported ratio is below 1, we bring it up to 1, ",
+                              "reflecting the assumption that underlying medical conditions do not lessen suseptibility to COVID-19."),
+                      tags$li("To adjust for ICU risk, we use unadjusted odds ratios from CDCC (for all but obesity). ",
+                              "To avoid overinflating risk due to the odds ratios being unadjusted, we adjust only for ",
+                              "the two highest risk conditions inputted by the user. ",
+                              "To adjust for obesity, we use the mutually adjusted odds ratio from Simonnet et al."),
+                      tags$li("To adjust death risk, we use mutually adjusted odds ratios from OpenSafely. ",
```

Comment:
@e-lipman, please add a hyperlink to OpenSAFELY study. Note the capitalized and uncapitalized letters in the study name.